internal/cli/local/setup/run.go

```diff
@@ -58,47 +60,69 @@ func readConfiguration(jsonConfig string) (*chart.Configuration, error) {
 	return config, nil
 }
 
-func runInstallOrUpgrade(cmd *cobra.Command, args []string) error {
-	if installCmdOpts.remoteVersion != "" && !installCmdOpts.remoteDownload {
-		return fmt.Errorf("%w: --remote-version can only be used with --remote-download", utils.ErrValidationFailed)
+func runSetup(cmd *cobra.Command, args []string) error {
```

Comment:
I'm confused why you are making overlay on top of chart installer and not just making another standalone command which depends on k3s/chart installer utilities

Reply:
you wanted to keep old commands? I think main point Anton was aksing was to have single command for all install, and single command for upgrade

Reply:
I thought just to make a low-level set of commands and high-level, but if you want to merge them up - okay

Reply:
I think about adding sub-command later like install just new chart, but it needs to be discussed how we see it